content/docs/setup/kubernetes/quick-start-gke-dm/index.md

```diff
@@ -208,7 +209,30 @@ http://localhost:9090/graph
 
 For more details, see [About the Prometheus Add-on](/docs/tasks/telemetry/querying-metrics/#about-the-prometheus-add-on).
 
-### ServiceGraph
+### Kiali and ServiceGraph
+
+Set up a tunnel to Kiali:
+
+{{< text bash >}}
+$ kubectl -n istio-system port-forward $(kubectl -n istio-system get pod -l app=kiali -o jsonpath='{.items[0].metadata.name}') 20001:20001
+{{< /text >}}
+
+You should see the Bookinfo service topology at
+
+{{< text plain >}}
+http://localhost:20001/kaili
+{{< /text >}}
+
+The default username/password for the console is `admin`/`mysecret`.
+
+{{< image width="100%" ratio="53.33%"
+    link="./dm_kiali.png"
+    caption="Kiali"
+    >}}
+
+For more details, see [About the Kiali Add-on](/docs/tasks/telemetry/kiali/).
+
+OR
```

Comment:
This is not the appropriate way to do it. Please add subheadings for each Kiali and ServiceGraph.
```suggestion
#### ServiceGraph
```